test/functional/helpers.lua

```diff
@@ -501,9 +501,20 @@ function module.source(code)
 end
 
 function module.set_shell_powershell()
+  local shell = iswin() and 'powershell' or 'pwsh'
+  if not module.eval('executable("'..shell..'")') then
+    error(shell..' is not executable')
+  end
+  local aliases = iswin() and {'cat', 'sleep'} or {}
+  local cmd = ''
+  for _, alias in ipairs(aliases) do
+    cmd = cmd .. 'Remove-Item -Force alias:' .. alias .. ';'
+  end
```

Comment:
could save a lot of code by just hardcoding two different `let &shellcmdflag ...` strings. But I don't want to block this.

Reply:
I'll shortened this to one-line. TIL that `Remove-Item` can accept an array to delete multiple aliases.